cmd/create/accountroles/validation.go

```diff
@@ -6,25 +6,26 @@ import (
 	"github.com/openshift/rosa/pkg/arguments"
 )
 
-func validateSharedVpcInputs(hostedCp bool, vpcEndpointRoleArn string,
-	route53RoleArn string) (bool, error) {
-	if hostedCp {
-		if vpcEndpointRoleArn != "" && route53RoleArn == "" {
-			return false, errors.UserErrorf(
-				arguments.MustUseBothFlagsErrorMessage,
-				route53RoleArnFlag,
-				vpcEndpointRoleArnFlag,
-			)
-		}
+func validateSharedVpcInputs(vpcEndpointRoleArn string, route53RoleArn string) (bool, error) {
```

Comment:
we should probably have a separate package like roles/validation because the behavior seems to be shared